sharding-scaling/sharding-scaling-core/src/main/java/org/apache/shardingsphere/shardingscaling/core/execute/executor/writer/SqlBuilder.java

```diff
@@ -66,6 +66,23 @@ public String load(final String key) {
                     }
                 });
     }
+    
+    private void initIdentifierQuoteString(final String databaseType) {
+        switch (databaseType) {
+            case "MySQL":
+                leftIdentifierQuoteString = "`";
+                rightIdentifierQuoteString = "`";
+                break;
+            case "PostgreSQL":
+                leftIdentifierQuoteString = "\"";
+                rightIdentifierQuoteString = "\"";
+                break;
+            default:
+                leftIdentifierQuoteString = "";
+                rightIdentifierQuoteString = "";
+                break;
+        }
+    }
```

Comment:
Similar problem as AbstractJdbcReader.
This can abstract a protect method, and let subClass to implement.

Or create a new interface to handle this situation.
each type database has implement, SQLBuilder only call interface.